app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -228,6 +228,7 @@ class StatePlayerRecyclerViewAssembler private constructor(
         )
       }
     } else if (ephemeralState.stateTypeCase == EphemeralState.StateTypeCase.COMPLETED_STATE) {
+      isCorrectAnswer.set(true)
```

Comment:
Please add a brief comment here explaining this for future readers (e.g. something like "Ensure the answer is marked in situations where that's guaranteed (e.g. completed state) so that the UI always has the correct answer indication, even after configuration changes." or similar).

This helps provide context for others since otherwise this seems almost random despite it having very specific intention.

Reply:
done